README.md

```diff
@@ -1,14 +1,14 @@
 [![Build Status](https://travis-ci.org/gin-gonic/website.svg?branch=master)](https://travis-ci.org/gin-gonic/website)
 
-# Gin website
+# Gin website 
 
-Welcome! This repository houses all of the assets required to build the Gin website and documentation. We're very pleased that you want to contribute! The website are hosted at https://gin-gonic.com.
+Welcome! This repository houses all of the assets required to build the Gin website and documentation. We're very pleased that you want to contribute! The websites are hosted at https://gin-gonic.com.
```

Comment:
Instead of "The websites are hosted at https://gin-gonic.com" the better alternative in my opinion to make the most grammatical sense would be to make it "The website is hosted at https://gin-gonic.com"

All of the other changes do not fix anything that is not accurate or formatted incorrectly.